.circleci/config.yml

```diff
@@ -166,6 +166,10 @@ commands:
   build_docker_image:
     description: "Builds a Docker image"
     steps:
+      - when:
+            equal: [ /^v.*/, << pipeline.git.tag >> ]
+            steps:
+              - *notify_slack_on_release_start
```

Comment:
Don't really like this approach, as it might not be in sync with the trigger filters.
Will look at other options.